SwiftCheck/Property.swift

```diff
@@ -290,13 +326,15 @@ public struct TestResult {
 	let callbacks		: [Callback]
 	/// Indicates that any further testing of the property should cease.
 	let abort			: Bool
+	///
+	let quantifier		: Quantification
 
 	/// Destructures a test case into a matcher that can be used in switch statement.
-	public func match() -> TestResultMatcher {
-		return .MatchResult(ok: ok, expect: expect, reason: reason, theException: theException, labels: labels, stamp: stamp, callbacks: callbacks, abort: abort)
+	internal func match() -> TestResultMatcher {
```

Comment:
These are all `internal` by default, no? (To be clear, I am not intending to be prescriptive re: code style, just curious why it should be explicit on the function & `init` but not the `let`s above.)